juniper_official/README.md

```diff
@@ -1 +1,16 @@
-Rules by juniper employees go here
+# Healthbot default KPIs
+#
+ 
+Contains readily consumable healthbot playbooks and rules which are specific to chassis, linecard, system, interface and protocols
+key performance indicators(KPIs). KPI rules collects the statistics from network devices then analyzes the data and act.
+
+KPI playbook is set of rules, each rule is defined with set of KPIs.
+
+Playbooks for chassis, linecard, system, interface and protocols(bgp, lldp, rib) are available in respective folders.
+
+Rules are defined with default variable values which can be changed while deploying playbook.
+
+
+## Usage
+
+Apply the playbook to device-group under playbooks section using healthbot GUI.
```

Comment:
Please change all instances of healthbot to HealthBot.
Also change the sentence to:
Apply the playbook to device-group under the playbooks section using the HealthBot GUI.

Reply:
Addressed